files/validator.py

```diff
@@ -87,7 +77,8 @@ def _validate_lists_in_mapping_file(self, mapping_vars):
             mapping_vars, self.aff_label_map) and self._is_list(
             mapping_vars, self.network_map)
 
-    def _is_list(self, mapping_vars, mapping):
+    @staticmethod
+    def _is_list(mapping_vars, mapping):
```

Comment:
staticmethod should be called from the class itself and not from an instantiated object/self-object, I think it is better to leave it as is.

BTW, is there a issue with those methods or it is just for readability?

Reply:
Reverting.
My IDE complained about it, so I though that if I'm touching this code I will improve it.
I turned off this warning for the future.